features/user_associated_with_multiple_accounts.md

```diff
@@ -0,0 +1,27 @@
+# User associated with multiple accounts
+
+A user can be associated with multiple ImageKit accounts using the same email address. A previously registered user can be added to a different account. Read how to add a user to an account [here](./user-access-management/README.md).
+
+The login flow for a multi-account user is as follows:
+
+You will be asked to put in your email and password (any valid password for one of the accounts) at the login screen.
+
+![Login screen](../.gitbook/assets/login_screen.png)
+
+Then you will be redirected to the following page and asked to put in a one-time password (OTP) sent to the associated email ID.
```

Comment:
Then you'll be sent to the next page and prompted to enter a one-time password (OTP) that was sent to the associated email address.